DistroLauncher/upgrade_policy.cpp

```diff
@@ -54,26 +54,21 @@ void SetDefaultUpgradePolicyImpl()
     const fs::path log{L"/var/log/upgrade-policy-changed.log"};
     const fs::path policyfile{L"/etc/update-manager/release-upgrades"};
 
-    if (fs::exists(log)) {
+    if (fs::exists(Oobe::WindowsPath(log))) {
         return;
     }
 
     std::wstring regex = L"s/Prompt=lts/Prompt=" + GetUpgradePolicy() + L'/';
     std::wstringstream sed;
     sed << L"sed -i " << std::quoted(regex) << L' ' << std::quoted(policyfile.wstring());
+    sed << L" && date --iso-8601=seconds" << L" > " << std::quoted(log.wstring());
 
```

Comment:
I think the command needs to be `bash -ec \' ... \' `. Did you test that this composition work in WSL? I'm afraid it would interpret the command composition as a single executable path, which would obviously fail due file not found.

Reply:
Oh I replaced it to your sugestion before testing this, I pushed too early.